Source/Tailwind.Traders.Web/Controllers/PersonalizerController.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.CognitiveServices.Personalizer;
+using Microsoft.Azure.CognitiveServices.Personalizer.Models;
+using System;
+using System.Collections.Generic;
+
+namespace Tailwind.Traders.Web.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class PersonalizerController : Controller
+    {
+        private PersonalizerClient _personalizerClient;
+        private static readonly string ApiKey = Environment.GetEnvironmentVariable("PERSONALIZER_RESOURCE_KEY");
+        private static readonly string ServiceEndpoint = Environment.GetEnvironmentVariable("PERSONALIZER_RESOURCE_ENDPOINT");
+
+        public PersonalizerController()
+        {
+            _personalizerClient = GetPersonalizerClient(ApiKey, ServiceEndpoint);
+        }
+
+        [HttpGet("Rank")]
+        public IActionResult Get()
+        {
+            RankRequest request = GetRankRequest();
+            RankResponse response;
+            try
+            {
+                response = _personalizerClient?.Rank(request);
```

Comment:
If `_personalizerClient` is `null`, won't that cause the response object to be `null`? This will have downstream effects that are unintended.

Reply:
Added logic to do nothing when response statusTest==="No Content"